api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -118,4 +118,30 @@
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
     
+	<changeSet id="20180214-1000" author="Kelechi+iheanyi" dbms="mysql">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="condition"/></not>
+		</preConditions>
+		<comment>
+			Add condition table
+		</comment>
+		<createTable tableName="condition">
+			<column name="condition_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
```

Comment:
Is there a reason why you used capital letter for VARCHAR instead of small? 😊

Reply:
No reason at all. However, I found out it is case insensitive. After going through the liquibase website, I notice their examples use lowercase mostly. Hence, for conformity sake, I'm effecting this correction.

Reply:
Done